python/tvm/target/target.py

```diff
@@ -220,19 +220,24 @@ def intel_graphics(model="unknown", options=None):
     return Target(" ".join(["opencl"] + opts))
 
 
-def micro(hardware="unknown", options=None):
+def micro(model="unknown", options=None):
     """Returns a microTVM target.
 
     Parameters
     ----------
-    hardware : str
-        Canonically identifies the target device; typicaly one of cortex-mX, or a specific SoC model
-        when that model has been tested to work with microTVM.
+    model : str
+        Canonically identifies the target device. This is typically a CPU or board level name (other
+        flags such as -mcpu identify the ISA).
     options : str or list of str
         Additional options
     """
-    trans_table = {"host": ["-mcpu=native"]}
-    opts = _merge_opts(trans_table[hardware] + ["-runtime=c", "--system-lib"], options)
+    trans_table = {
```

Comment:
Someday we're going to have to / want to work with targets other then cpu. Having a per board hard code here makes me a little uneasy. Should we  be open to a board / -mcpu setting passed in to override or add to the trans_table?

Reply:
you can always pass in a specific CPU by manually creating a target: `tvm.target.create("c -mcpu=cortex-m4")`. I do agree that this might become unworkable in the long run--i'm mostly doing this here since it was past convention. for now i don't want to mess with that convention, but i'm open to it in the future. feel free to suggest too if you have a better idea!